### PR TITLE
Optimizes spm_multilineIndent of StringExtensions

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

File: Sources/TSCUtility/StringExtensions.swift [hunk 1]
```diff
@@ -96,9 +96,12 @@ extension String {
...
-            .split(separator: "\n")
-            .map{ String(repeating: " ", count: count) + $0 }
-            .joined(separator: "\n")
+        let indent = String(repeating: " ", count: count)
+        let fullRange = self.startIndex ..< self.endIndex
+        var ret = ""
+        self.enumerateSubstrings(in: fullRange, options: [.byLines, .substringNotRequired]) { _, _, enclosingRange, _ in
+            ret += indent + self[enclosingRange]
+        }
+        return ret
```
Comment (aciidb0mb3r) [inline Sources/TSCUtility/StringExtensions.swift]: Thanks for looking at this! Have you done any performance comparison with this change? I wonder if bridging to ObjC hurts performance on macOS. 

cc  @milseman

Comment (valeriyvan) [reply Sources/TSCUtility/StringExtensions.swift]: Actually you were right! Implicit bridging made fix ~25 times slower than original implementation on file of 2000 lines. So I rolled back original implementation and just applied slight optimisation with moving string creation out of closure (~9% gain on a 2000 lines file).

Comment (aciidb0mb3r) [reply Sources/TSCUtility/StringExtensions.swift]: Great, thank you!

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

**Pull request merged**